src/scss/02_atoms/_formField.scss

```diff
@@ -43,8 +43,11 @@
   }
 
   & + & {
+    margin-top: 10px;
+
     @include mq(small) {
       margin-left: 10px;
+      margin-top: initial;
```

Comment:
Non usare `initial` in quanto non è uguale in tutti i browsers ed ognuno di loro lo implementa secondo il motore di rendering proprietario. 
Meglio fare: 
```
& + & {
   @include mqDown(small) {
      margin-top: 10px;
   }
   
   @include mq(small) {
      margin-left: 10px;
   }   
 
}
```